examples/CURL/cmake/Hunter/config.cmake

```diff
@@ -0,0 +1,2 @@
+hunter_config(OpenSSL VERSION 1.1.0)
```

Comment:
Just for your info it was not tested. Default version is `1.0.2*`: https://github.com/ruslo/hunter/blob/master/cmake/configs/default.cmake . I have issues with 1.1.0 with building on Windows.